h2o-py/h2o/h2o.py

```diff
@@ -2211,6 +2211,20 @@ def print_mojo(mojo_path, format="json", tree_index=None):
     else:
         raise H2OError("Unable to print MOJO: %s" % output)
 
+def set_system_property(property, value):
+    """
+    Set the specified system property to specified value 
+
+    :param property: system property to be set
+    :param value: value to be set
+
+    :example:
+
+    >>> h2o.set_system_property("sys.ai.h2o.algos.evaluate_auto_model_parameters", "false")
```

Comment:
I'm not sure we should provide direct access to all system properties like that, I would rather limit to some feature flags for now: 
1. there may be security concerns.
2. those properties are not discoverable by the user.
3. validation concern: if user makes a typo in the property name, the property will be set and user won't notice.

My suggestion:
- backend always adds the "sys.ai.h2o." prefix. This will strictly limit the kind of properties that the clients can change (it may still be too broad).
- expose a map on the client with just the flags we want to expose so that the user knows what can be switched on/off.

 I would limit this to feature flags for now, for example for the Py client
```
feature_flags = dict(
   enable_evaluation_of_auto_model_parameters = ("algos.evaluate_auto_model_parameters", True),
   enable_xgboost_in_automl_multinode = ("automl.xgboost.multinode.enabled", True)
)

def set_feature_flag(feature, enable=None):
   assert feature in feature_flags
   key, def_value = feature_flags[feature]
   value = def_value if enable is None else enable
   value = "true" if value else "false"
   api("POST /3/SetSystemProperty", data=dict(property=key, value=value))
```

Now user can do 
```
h2o.set_feature_flag("enable_evaluation_of_auto_model_parameters", False)
And it will fail if there's a typo in the property.
```

I also have a concern with the name: I think we should specify that those properties or feature flags are only for the backend, otherwise it may confuse some users as some client features can also be switched using the `h2oconfig` file.
My suggestion would be to expose this only in the `h2o.backend` module for clarity.

Also, there may be concerns that I didn't notice yet (@michalkurka ?), so I wouldn't mind if you prefer to implement this in a separate PR.

Reply:
Hm yes, this approach with `assert feature in feature_flags` will give us much better control over what is being set by this. With separate PR... I think that the switcher of the new feature should be a part of the PR with the new feature, so I would do this here at least in some very specific form (only `enable_evaluation_of_auto_model_parameters `flag in feature_flags). If some concerns will raise afterwards, it can still be perfected/extended for other features/etc in subsequent PR

Reply:
6fc83f2